### PR TITLE
Chore: Rename "marketplace" app to "catalog"

### PR DESCRIPTION
**Pull request by ryantxu**
The plugin "marketplace" is an odd name for the list of plugins -- we do not have a store or any plans to make selling plugins a standard offering.

This PR renames the plugin and docs to "catalog" rather than "marketplace" -- it looks like the page layouts need some :heart: before final release, but that should happen after this PR is merged  (or some other appropriate name)

### CODE REVIEW

File: docs/sources/plugins/catalog.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+weight = 1
++++
+
+# Plugin catalog
+
+Catalog for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.
+
+[screenshot placeholder]
+
+To use the Catalog for Grafana, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#catalog_app_enabled" >}}).
```
Comment (oddlittlebird) [inline docs/sources/plugins/catalog.md:13]: Is this true? In beta1, the app was right there without me enabling anything.

File: docs/sources/plugins/catalog.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
++++
+title = "Plugin catalog"
+aliases = ["/docs/grafana/latest/plugins/catalog/"]
+weight = 1
++++
+
+# Plugin catalog
```
Comment (oddlittlebird) [inline docs/sources/plugins/catalog.md:7]: Any ETA on adding content for this?

File: docs/sources/administration/configuration.md [hunk 3]
```diff
@@ -1467,17 +1467,17 @@ Set to `true` if you want to test alpha plugins that are not yet ready for gener
 
 Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loaded even if they lack a valid signature.
 
-### marketplace_url
+### catalog_url
 
 Custom install/learn more url for enterprise plugins. Defaults to https://grafana.com/grafana/plugins/.
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:1472]:
```suggestion
Custom install/learn more URL for enterprise plugins. Defaults to https://grafana.com/grafana/plugins/.
```

Comment (oddlittlebird) [reply docs/sources/administration/configuration.md:1472]: Is it just enterprise plugins? Or is it all plugins?

File: docs/sources/administration/configuration.md [hunk 4]
```diff
@@ -1467,17 +1467,17 @@ Set to `true` if you want to test alpha plugins that are not yet ready for gener
...
 
-### marketplace_app_enabled
+### catalog_app_enabled
 
 > **Note:** Available in Grafana 8.0 and later versions.
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:1476]:
```suggestion
```

**Review by torkelo — APPROVED** [review]
Looks good

File: docs/sources/plugins/catalog.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Catalog for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.
```
Comment (osg-grafana) [inline docs/sources/plugins/catalog.md:9]: There is an instance of "Marketplace" on line 9.

**Comment by osg-grafana** [comment]
cc @jhunthrop because this touches GE plugins

**Pull request merged**